python/ray/worker.py

```diff
@@ -46,12 +46,16 @@
 LOCAL_MODE = 2
 PYTHON_MODE = 3
 
+# This definition should be the same with gcs.fbs.
+LANGUAGE_PYTHON = 0
+LANGUAGE_CPP = 1
+LANGUAGE_JAVA = 2
```

Comment:
Define them in ray_constant. Then if we use them in other files, we won't introduce circular imports.